File: 3. Find the ASCII value of a character/code.cpp
#include <iostream>
using namespace std;

int asciiValue(char c) {
    return int(c);
}

int main() {
    char input_char;
    cout << "Enter a character: ";
    cin >> input_char;
    cout << "ASCII value: " << asciiValue(input_char) << endl;
    return 0;
}

File: 3. Find the ASCII value of a character/code.java
import java.util.Scanner;

public class Main {
    public static int asciiValue(char c) {
        return (int) c;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a character: ");
        char inputChar = scanner.next().charAt(0);
        System.out.println("ASCII value: " + asciiValue(inputChar));
    }
}
